sdk/identity/azure-identity/src/main/java/com/azure/identity/AadCredentialBuilderBase.java

```diff
@@ -77,4 +80,31 @@ public T executorService(ExecutorService executorService) {
         this.identityClientOptions.setExecutorService(executorService);
         return (T) this;
     }
+
+    /**
+     * For multi-tenant applications, specifies additional tenants for which the credential may acquire tokens.
+     * Add the wildcard value "*" to allow the credential to acquire tokens for any tenant on which the application is installed.
```

Comment:
Is it intentional that you don't have "If no value is specified for TenantId this option will have no effect, and the credential will acquire tokens for any requested tenant" for this JavaDoc?

Reply:
yes, because this API from here flows to credentials Client Secret Credential, Client Certificate Credential, OBO Credential, Client Assertion Credential etc. where tenant id parameter is required.